stream_alert_cli/test.py

```diff
@@ -89,6 +89,9 @@ def __init__(self, context, config, print_output):
         # Create a cache map of parsers to parser classes
         self.parsers = {}
 
+        # Patch the tmp shredding as to not slow down testing
```

Comment:
Well, you also definitely don't want to shred peoples /tmp directories when running unit tests :)